config.yml

```diff
@@ -1,51 +1,24 @@
 experiment_name: my_example
-n_repeats: 5
+n_repeats: 1
 scenario_params_list:
  - dataset_name:
-     - 'mnist'
-     - 'cifar10'
+     - 'titanic'
    partners_count:
      - 3
-   amounts_per_partner: 
+   amounts_per_partner:
      - [0.4, 0.3, 0.3]
-   samples_split_option: 
+   samples_split_option:
      - ['basic', 'random']
-     - ['basic', 'stratified']
-     - ['advanced', [[4, 'shared'], [6, 'shared'], [4, 'specific']]]
    multi_partner_learning_approach:
-     - 'fedavg'
      - 'seq-pure'
-     - 'seq-with-final-agg'
-     - 'seqavg'
-   aggregation_weighting: 
-     - 'data_volume' 
-     - 'uniform'
-   epoch_count: 
-     - 38
+     - 'fedavg'
+   aggregation_weighting:
+     - 'data_volume'
+   epoch_count:
+     - 10
    methods:
      - ["Shapley values", "Independent scores", "TMCS"]
-   minibatch_count: 
-     - 20
+   minibatch_count:
+     - 5
    gradient_updates_per_pass_count:
      - 8
```

Comment:
I think that `8` is a lot with that few samples in the dataset...

Reply:
I've changed it to 3, it seems to provide a small improvement in accuracy, but a nice speedup in computing time, which is nice.